### PR TITLE
feat(uuid): uuid v3

### PR DESCRIPTION
**Pull request by lino-levan**
Discussed a bit on discord, no one objected at the time.

### CODE REVIEW

File: uuid/v3.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+/**
+ * Generate a RFC4122 v3 UUID (MD5 namespace).
+ *
+ * @example
+ * ```js
+ * import { generate } from "https://deno.land/std@$STD_VERSION/uuid/v3.ts";
+ *
+ * const NAMESPACE_URL = "6ba7b810-9dad-11d1-80b4-00c04fd430c8";
```
Comment (kt3k) [inline uuid/v3.ts:35]: This UUID looks like NAMESPACE_DNS according to RFC4122 appendix C ref. https://datatracker.ietf.org/doc/html/rfc4122#appendix-C (NAMESPACE_URL is `6ba7b811-9dad-11d1-80b4-00c04fd430c8` instead. You can also check it with python's `uuid` standard module.

```shellsession
$ python3
Python 3.11.2 (main, Feb 16 2023, 02:51:42) [Clang 14.0.0 (clang-1400.0.29.202)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import uuid
>>> uuid.NAMESPACE_DNS
UUID('6ba7b810-9dad-11d1-80b4-00c04fd430c8')
>>> uuid.NAMESPACE_URL
UUID('6ba7b811-9dad-11d1-80b4-00c04fd430c8')
```

File: uuid/v3.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ *
+ * const uuid = await generate(NAMESPACE_URL, new TextEncoder().encode("python.org"));
+ * uuid === "886313e1-3b8a-5372-9b90-0c9aee199e5d" // true
```
Comment (kt3k) [inline uuid/v3.ts:38]:
```suggestion
 * uuid === "6fa459ea-ee8a-3ca4-894e-db77e160355e" // true
```

```shellsession
$ cat a.js 
import { generate } from "./uuid/v3.ts";
const NAMESPACE_URL = "6ba7b810-9dad-11d1-80b4-00c04fd430c8";
const uuid = await generate(NAMESPACE_URL, new TextEncoder().encode("python.org"));
console.log(uuid);
$ deno run a.js 
6fa459ea-ee8a-3ca4-894e-db77e160355e
```

**Comment by lino-levan** [comment]
Ah sorry for the sloppy PR, I mean to open it as a draft. Will look into changes this weekend.

File: uuid/v3.ts (hunk 1 above)
Comment (lino-levan) [reply uuid/v3.ts]: Just wanted to note that I got that example from the UUIDv5 implementation... this PR fixes this mistake over there as well.

File: uuid/v5.ts [hunk 3]
```diff
@@ -31,10 +31,10 @@ export function validate(id: string): boolean {
  * ```js
  * import { generate } from "https://deno.land/std@$STD_VERSION/uuid/v5.ts";
  *
- * const NAMESPACE_URL = "6ba7b810-9dad-11d1-80b4-00c04fd430c8";
+ * const NAMESPACE_URL = "6ba7b811-9dad-11d1-80b4-00c04fd430c8";
```
Comment (kt3k) [inline uuid/v5.ts:34]: 👍

**Pull request merged**